cmd/rm.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var rmCmd = &cobra.Command{
+	Use:   "rm",
+	Short: "Remove a project from your dependency.",
+	Long:  `Remove a project from your dependency.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		viper.Unmarshal(&cfg)
+		for _, arg := range args {
+			for i, ele := range cfg.Projects {
+				if arg == ele {
+					cfg.Projects = remove(cfg.Projects, i)
```

Comment:
This is likely me being unsure about go semantics but is assigning during iteration maybe going to cause issues?

Reply:
If I'd iterated backwards, I probably would have been fine. Though someone is going to ask about this again in the future. Changed to make a copy of the slice.

This command will probably be depreciated in the near future anyway.